src/components/AddressForm/index.tsx

```diff
@@ -114,10 +115,10 @@ const AddressForm = ({ type, errorMessages }: AddressFormProps) => {
         fieldName={`${type}-zip`}
         isRequired={isRequired}
         errorState={errors}
-        minLength={MAXLENGTHZIP}
+        // minLength={MAXLENGTHZIP}
```

Comment:
Is this supposed to be commented out?  Also, is it supposed to be `MINLENGTHZIP`, not `MAX`?

Reply:
It was giving me trouble when re-validating addresses but yes, you're right, it was suppose to be `MINLENGTHZIP`. This is fixed in #83 (fixed it before seeing this comment).